js/src/components/paid-ads/ads-campaign.js

```diff
@@ -25,19 +25,23 @@ import FaqsSection from './faqs-section';
 /**
  * Renders the container of the form content for campaign management.
  *
- * Please note that this component relies on an CampaignAssetsForm's context and custom adapter,
- * so it expects a `CampaignAssetsForm` to existing in its parents.
+ * Please note that this component relies on a CampaignAssetsForm's context and custom adapter,
+ * so it expects a `CampaignAssetsForm` to exist in its parents.
  *
  * @fires gla_documentation_link_click with `{ context: 'create-ads' | 'edit-ads' | 'setup-ads', link_id: 'see-what-ads-look-like', href: 'https://support.google.com/google-ads/answer/6275294' }`
  *
  * @param {Object} props React props.
  * @param {Campaign} [props.campaign] Campaign data to be edited. If not provided, this component will show campaign creation UI.
  * @param {() => void} props.onContinue Callback called once continue button is clicked.
+ * @param {() => void} props.isLoading Callback called once continue button is clicked.
```

Comment:
@kt-12 This is incorrect. `isLoading` is a `boolean`. Can you update please?

Reply:
`@param {() => void}` to `@param {() => boolean}` Does the text seems alright to you?

Reply:
I see you've updated the comment with the correct syntax.